app/assets/stylesheets/refills/_navigation.scss

```diff
@@ -0,0 +1,85 @@
+.navigation {
+  $navigation-background: $blue;
+  $navigation-color: darken($base-background-color, 20%);
+  $navigation-color-hover: $base-background-color;
+  $navigation-height: 60px;
+  $horizontal-bar-mode: $large-screen;
+
+  background-color: $navigation-background;
+  border-bottom: 1px solid darken($navigation-background, 10);
+  min-height: $navigation-height;
+  width: 100%;
+  z-index: 999;
+
+  &-wrapper {
+    @include clearfix;
+    @include outer-container;
+    position: relative;
+    z-index: 9999;
+  }
+
+  nav {
+    float: none;
+    min-height: $navigation-height;
+    z-index: 9999999;
+
+    @include media ($horizontal-bar-mode) {
+      float: left;
+    }
+  }
+
+  &-menu {
+    clear: both;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets